config/production/vars.json

```diff
@@ -3,5 +3,6 @@
   "COFFEE_ROLE_ID": "936899047816589354",
   "NOTIF_CHANNEL_ID": "872305316522508329",
   "ANNOUNCEMENTS_CHANNEL_ID": "669294029804011540",
-  "OFFICE_CHANNEL_ID": "943680816880447488"
+  "OFFICE_CHANNEL_ID": "943680816880447488",
```

Comment:
if we are using office_status_channel_id, we don't need office_channel_id anymore